app/code/Magento/BundleGraphQl/README.md

```diff
@@ -1,4 +1,26 @@
 # BundleGraphQl
 
-**BundleGraphQl** provides type and resolver information for the GraphQl module
+**Magento_BundleGraphQl** module extends **Magento_GraphQl** and **Magento_Bundle** modules by provides type and resolver information for the GraphQl module
 to generate bundle product information.
+
+## Installation details
+
+Before disabling or uninstalling this module, note that the following modules depends on this module:
+
+- Magento_GraphQl
+- Magento_Bundle
+- Magento_QuoteGraphQl
+- Magento_CatalogGraphQl
+- Magento_Store
+
+For information about module installation in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```

Comment:
```suggestion
For information about enabling or disabling a module in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```

Reply:
Hi, @rogyar !
Thank you for review..
Changes was added.